src/main/java/uk/gov/hmcts/reform/pip/account/management/service/PublicationService.java

```diff
@@ -40,11 +41,43 @@ public class PublicationService {
      */
     public boolean sendNotificationEmail(String emailAddress, String forename, String surname) {
         JSONObject jsonObject = new JSONObject();
-        jsonObject.put("email", emailAddress);
+        jsonObject.put(EMAIL, emailAddress);
         jsonObject.put("forename", forename);
         jsonObject.put("surname", surname);
         try {
             log.info(webClient.post().uri(url + "/notify/created/admin")
+                         .body(BodyInserters.fromValue(jsonObject)).retrieve()
+                         .bodyToMono(String.class).block());
+            return true;
+        } catch (WebClientException ex) {
+            log.error(String.format("Request failed with error message: %s", ex.getMessage()));
+            return false;
+        }
+    }
+
+    public boolean sendNotificationEmailForSetupMediaAccount(String emailAddress, String fullName) {
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put(EMAIL, emailAddress);
+        jsonObject.put("isExisting", false);
+        jsonObject.put("fullName", fullName);
+        try {
+            log.info(webClient.post().uri(url + WELCOME_EMAIL_URL)
+                .body(BodyInserters.fromValue(jsonObject)).retrieve()
+                .bodyToMono(String.class).block());
+            return true;
+
+        } catch (WebClientException ex) {
+            log.error(String.format("Request failed with error message: %s", ex.getMessage()));
+            return false;
+        }
+    }
```

Comment:
please remember to check code that already exists, this code is directly under the code you added, the only thing you needed to add was the full name field, please remove and add to the body of the method that already exists

Reply:
Done

Reply:
im still seeing it, you need to delete the method `sendMediaNotificationEmail` as it already exists as the first method in this class, delete that enture method and use the one that already exists called `sendNotificationEmail`